### PR TITLE
Fix some dosctring RT02 error

### PR DESCRIPTION
**Pull request by luke396**
- [ ] closes #xxxx (Replace xxxx with the GitHub issue number)
- [ ] [Tests added and passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#writing-tests) if fixing a bug or adding a new feature
- [ ] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).
- [ ] Added [type annotations](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#type-hints) to new arguments/methods/functions.
- [ ] Added an entry in the latest `doc/source/whatsnew/vX.X.X.rst` file if fixing a bug or adding a new feature.

### CODE REVIEW

**Comment by luke396** [comment]
About #49968

File: pandas/core/indexes/multi.py [hunk 1]
```diff
@@ -2862,10 +2862,11 @@ def get_loc_level(self, key, level: IndexLabel = 0, drop_level: bool = True):
 
         Returns
         -------
-        loc : A 2-tuple where the elements are:
-              Element 0: int, slice object or boolean array
-              Element 1: The resulting sliced multiindex/index. If the key
-              contains all levels, this will be ``None``.
+        tuple
```
Comment (phofl) [inline pandas/core/indexes/multi.py:2865]: Can you render this and check how it looks?

Comment (luke396) [reply pandas/core/indexes/multi.py:2865]: > Can you render this and check how it looks?

Yes, that is not pretty, and I make some changes, it looks like this now.

image

**Review by MarcoGorelli — APPROVED** [review]
Looks good to me, over to @phofl

**Review by phofl — APPROVED** [review]

**Comment by phofl** [comment]
thx @luke396

**Pull request merged**